opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -188,12 +179,15 @@ namespace Opm {
             const double gravity = detail::getGravity(geo_.gravity(), UgGridHelpers::dimensions(grid_));
             const std::vector<double> pv(geo_.poreVolume().data(), geo_.poreVolume().data() + geo_.poreVolume().size());
             const std::vector<double> depth(geo_.z().data(), geo_.z().data() + geo_.z().size());
-            well_model_.init(fluid_.phaseUsage(), active_, &vfp_properties_, gravity, depth, pv, &rate_converter_);
+            // Wells are active if they are active wells on at least
+            // one process.
+            int wellsActive = localWellsActive() ? 1 : 0;
+            wellsActive = grid_.comm().max(wellsActive);
```

Comment:
The wellsActive variable is not used. To me it looks like you intended to also change the line below, to use the variable instead of localWellsActive()?